src/compiler.js

```diff
@@ -220,7 +220,23 @@ class Compiler {
         throw e
       }
 
-      let libs = path.join(baseDir, 'node_modules')
+      let libs
+      for (
+        let dir = path.resolve(baseDir);
+        // Terminate after the root dir or when we have found node_modules
+        dir !== undefined && libs === undefined;
+        // Continue with the parent directory, terminate after the root dir
+        dir = path.dirname(dir) === dir ? undefined : path.dirname(dir)
+      ) {
+        if (fs.existsSync(path.join(dir, 'node_modules'))) {
+          libs = path.join(dir, 'node_modules')
+        }
+      }
+
+      if (libs === undefined) {
+        throw Error(`could not find \`node_modules\` path`)
```

Comment:
Let's make this a little bit more helpful by saying
```
could not locate `node_modules` in parent directories of subgraph manifest `{}`
```

Reply:
Thanks, that's better.